semgrep/tox.ini

```diff
@@ -6,7 +6,7 @@ envlist = py37, py38, py39
 allowlist_externals = pipenv
 commands =
     pipenv install --sequential --dev
-    pipenv run pytest --ignore=tests/qa/test_public_repos.py -n auto -vv --tb=short
+    pipenv run pytest --ignore=tests/qa/test_public_repos.py -n auto -vv --tb=short {posargs}
```

Comment:
Please remove this line, we do want to run these QA tests, and this isn't relevant to the scope of the issue

Reply:
Will do.